src/components/layout/layout.module.scss

```diff
@@ -23,4 +26,15 @@
   @include breakpoints.respond-above(m) {
     --spacing: #{$spacing-layout-xs};
   }
+
+  .main {
+    grid-column: 1 / -1;
+
+    display: inherit;
+    grid-template-columns: inherit;
+
+    & > * {
+      grid-column: 2;
+    }
+  }
```

Comment:
Thanks for this! Now I don't have to figure out how to span the Hero section on all three columns 😄